### DISCUSSION TITLE
Project paused for no reason - eu-central-1

### DISCUSSION THREAD

**Discussion by 72perseid** [supabase/supabase#12395] [Questions]
Hello everyone!

I just wanted to know if anyone else's project on eu-central-1 was paused for no reason. It's taking eons to restore the database although I'm sure no one from our team paused it. Is this happening to anyone else or happened before? If there's anything I can do I'd be glad to lead the way.

**Comment by ruggi99** [comment]
Projects on the free tier are paused after 1 week of inactivity
